Formula/gq-cli.rb

```diff
@@ -0,0 +1,20 @@
+class GqCli < Formula
+  desc "Command-line interface for GQ"
+  homepage "https://goquant.io"
+  url "https://brickelldatabros1aboah.blob.core.windows.net/public/gq-cli-source-0.0.4.tar.gz"
+  sha256 "6e2b86cf57035bba045d7db9f1d28ae83283cec6b0ed4335083f320374b951ba"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root=#{prefix}", "--path=.", "--debug"
+  end
+
+  test do
+    system "#{bin}/gq"
+    system "#{bin}/gq", "--help"
+    system "#{bin}/gq", "config", "-p"
+    system "#{bin}/gq", "binance", "-h"
```

Comment:
The test still has room for improvement, as it seems they are still calling for help.

Also, can you please explain this:

```
$ gq
Welcome to GoQuant CLI (gq) please use the auth command to login.
Generated "/home/linuxbrew/gq-cli.config.yaml"

$ gq --help
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: reqwest::Error { kind: Request, url: Url { scheme: "http", cannot_be_a_base: false, username: "", password: None, host: Some(Domain("localhost")), port: Some(8080), path: "/login", query: None, fragment: None }, source: hyper::Error(Connect, ConnectError("tcp connect error", Os { code: 99, kind: AddrNotAvailable, message: "Cannot assign requested address" })) }', src/main.rs:31:62
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

$ cat /home/linuxbrew/gq-cli.config.yaml
username: ''
password: ''
live_api_url: wss://api.goquant.io/ws
static_api_url: http://localhost:8082
gq_auth_url: http://localhost:8080/login
```

Taken from `src/main.rs`:

```rust
/// Will panic if go_auth api is not live. Will handle errors later
fn authenticate<'a>(
    username: &'a str,
    password: &'a str,
) -> Result<reqwest::StatusCode, reqwest::Error> {
```

Reply:
The aim was to get a simple preview of this software up for a limited number of users, while causing the least amount of friction. Hence why we picked Homebrew. 

The reason why that fails is because the software was built in release, it is currently in a preview state after the first round of rapid prototyping. The team opted to ignore simple error handling as proper handling in rust is more methodical than other languages. 

A completed version: say 0.1.0 won't be relying on debug.